app/src/main/java/com/sopt/now/compose/ui/screens/login/LoginScreen.kt

```diff
@@ -31,12 +34,23 @@ import com.sopt.now.compose.ui.navigation.SignUpDestination
 @Composable
 fun LoginScreen(
     navController: NavHostController = rememberNavController(),
-    viewModel: LoginViewModel = viewModel(factory = LoginViewModel.Factory),
+    viewModel: LoginViewModel = viewModel(),
 ) {
     val context = LocalContext.current
     val uiState = viewModel.uiState.collectAsState()
```

Comment:
```suggestion
    val uiState by viewModel.uiState.collectAsState()
```
를 사용하신다면

```kotlin
    LaunchedEffect(uiState.message) {
```

아래 함수를 좀 더 간편하게 바꿀 수 있습니다.

Reply:
value 부분이 사라지는 게 확실히 깔끔하네요,, 적용해보겠습니다!